.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
     packages:
       - python2.4
 install:
-  - pip install tox PyYAML Jinja2 sphinx
+  - pip install tox PyYAML Jinja2 sphinx netaddr
```

Comment:
The test requirements are maintained in https://github.com/ansible/ansible/blob/devel/test/utils/tox/requirements.txt